setup.cfg

```diff
@@ -5,6 +5,8 @@ universal=1
 description_file = README.md
 
 [flake8]
+per-file-ignores=
+    jupyterfs/tests/test_pyfilesystem_manager.py:E302
```

Comment:
@telamonian here

Reply:
I don't want to have to add a special exception to the .cfg every time the linter disagrees with me able extra blank lines. Can we please make ignoring E302 a project-wide style?